framework/composer.json

```diff
@@ -54,7 +54,6 @@
     "require": {
         "php": ">=5.4.0",
         "ext-mbstring": "*",
-        "ext-mcrypt": "*",
```

Comment:
This is not true, even if ubuntu has it pre installed, if you compile PHP from source without any further arguments it is not included:

```
wget -O php-5.6.6.tar.bz2 http://de1.php.net/get/php-5.6.6.tar.bz2/from/this/mirror
tar xjvf php-5.6.6.tar.bz2
cd php-5.6.6/
./configure --prefix=/tmp/php/pjp-5.6.6-test

...
checking for OpenSSL support... no
...
```
so there should be a check in composer.json as there was before with mcrypt.